.sequelizerc

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env node
 const path = require('path')
 
+const migrationsDir =
+  process.env.IS_TESTING_DB_MIGRATION === "true" ?
+  "migrations-test" : "migrations";
+
 module.exports = {
   "config": path.resolve('./db/config.json'),
   "models-path": path.resolve('./src/models'),
   "seeders-path": path.resolve('./db/seeders'),
-  "migrations-path": path.resolve('./db/migrations')
+  "migrations-path": path.resolve(`./db/${migrationsDir}`),
```

Comment:
Would it be better to have it  "test/db"?

Reply:
I think I like `db/migrations` and `db/migrations-test`, feels like it is trivial, so ignoring it now.
Please let me know if you think it is quite important.